doc/src/manual/control-flow.md

```diff
@@ -717,17 +717,16 @@ Stacktrace:
 
 ### The `try/catch` statement
 
-The `try/catch` statement allows for `Exception`s to be tested for. For example, a customized
-square root function can be written to automatically call either the real or complex square root
-method on demand using `Exception`s :
+The `try/catch` statement allows for `Exception`s to be tested for, and for the graceful handling of things that may ordinarily break your application. For example, in the below code the function for square root would normally throw an exception and break your application. By placing a `try/catch` block around it we can mitigate that here. You may choose how you wish to handle this exception, whether logging it, return a placeholder value or as in the case below where we just printed out a statement. On line 740 and below there are more examples of handling exceptions with
```

Comment:
Also, the statement "On line 740" makes no sense in the documentation, as line numbers aren't visible there (and are subject to change in future PRs).

Reply:
And one more thing (which is my personal preference): saying "your application" may not be suitable for the Julia docs, simply because not every piece of Julia code that gets run falls under the standard definition of an application. Maybe it's ok, but I just wanted to mention it.